Examples/Cosmo/params.txt

```diff
@@ -0,0 +1,184 @@
+# See the wiki page for an explanation of the params!
+# https://github.com/GRChombo/GRChombo/wiki/Guide-to-parameters
+
+#################################################
+# Filesystem parameters
+
+verbosity = 0
+
+# location / naming of output files
+# output_path = "" # Main path for all files. Must exist!
+chk_prefix = Cosmo_
+plot_prefix = Cosmop_
+# restart_file = Cosmo_000000.3d.hdf5
+
+# HDF5files are written every dt = L/N*dt_multiplier*checkpoint_interval
+checkpoint_interval = 1000
+# set to 0 to turn off plot files (except at t=0 and t=stop_time)
+# set to -1 to never ever print plotfiles
+plot_interval = 500
+num_plot_vars = 7
+plot_vars = chi phi rho_scaled S_scaled K_scaled Ham Mom
+
+# subpaths - specific directories for hdf5, pout, extraction data
+# (these are created at runtime)
+hdf5_subpath = "hdf5"
+pout_subpath = "pout"
+data_subpath = "data"
+
+# change the name of output files
+# pout_prefix = "pout"
+print_progress_only_to_rank_0 = 1
+
+# ignore_checkpoint_name_mismatch = 0
+# write_plot_ghosts = 0
+
+#################################################
+# Initial Data parameters
+
+# Change the gravitational constant of the Universe!
+# Default is 1.0, for standard geometric units
+# Here we decouple the evolution so the scalar evolved on the
+# metric background without backreaction (this avoids the need
+# to solve the constaints)
+G_Newton = 1.
+
+# Scalar field initial data
+scalar_amplitude = 0.0777635
+scalar_width = 1.0 # not use
+scalar_mass = 1.0
+scalar_field_mode = 1.0
+
+# lineout params
+lineout_num_points = 10
+
+#################################################
+# Grid parameters
+
+# 'N' is the number of subdivisions in each direction of a cubic box
+# 'L' is the length of the longest side of the box, dx_coarsest = L/N
+# NB - If you use reflective BC and want to specify the subdivisions and side
+# of the box were there are no symmetries, specify 'N_full' and 'L_full' instead
+# NB - if you have a non-cubic grid, you can specify 'N1' or 'N1_full',
+# 'N2' or 'N2_full' and 'N3' or 'N3_full' ( then dx_coarsest = L/N(max) )
+# NB - the N values need to be multiples of the block_factor
+N_full = 128
+L_full = 1
+
+# Maximum number of times you can regrid above coarsest level
+max_level = 0 # There are (max_level+1) grids, so min is zero
+
+# Frequency of regridding at each level and thresholds on the tagging
+# Need one for each level except the top one, ie max_level items
+# Generally you do not need to regrid frequently on every level
+# in this example turn off regridding on all levels
+# Level Regridding: 0   1   2   3   4   5
+regrid_interval   = 0   0   0   0   0   0
+# regrid_threshold  = 0.5
+
+regrid_interval = 0 0 0 0 0 0
+
+# Max and min box sizes
+max_box_size = 8
```

Comment:
Normally 16 for both of these just to have a small ish number of boxes (512). Could also do 32 - depends on what we suggest for the wiki on how to load balance.